src/components/Organisation/Banner/index.tsx

```diff
@@ -73,21 +94,71 @@ const Banner: FC<IBanner> = ({ organisation }) => {
     return false;
```

Comment:
I think for `isFounder()` function, if we simply return the condition then it will return true or false on its own